src/main/scala/com/github/dotj/hellourld/Routes.scala

```diff
@@ -0,0 +1,95 @@
+package com.github.dotj.hellourld
+
+import akka.actor.typed.scaladsl.AskPattern._
+import akka.actor.typed.{ActorRef, ActorSystem}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.util.Timeout
+import com.github.dotj.hellourld.ShortLinkRegistry._
+
+import scala.concurrent.Future
+
+class Routes(registry: ActorRef[ShortLinkRegistry.Command])(implicit val system: ActorSystem[_]) {
```

Comment:
I renamed `ShortLinkRoutes.scala` -> `Routes.scala` and included both the shortlink management and redirect endpoints in one file for simplicity for now.